cmd/kubeadm/app/cmd/BUILD

```diff
@@ -1,10 +1,4 @@
-package(default_visibility = ["//visibility:public"])
```

Comment:
The problem with the bazel build (if i'm understanding bazel correctly) is that this line got deleted which makes the cmd:all-srcs target private. Since no default visibility is defined, the visibility will end up being private.

https://docs.bazel.build/versions/master/be/common-definitions.html#common.visibility